app/views/users/index.html.erb

```diff
@@ -1,12 +1,13 @@
 <% provide(:title, 'All users') %>
+<p></p>
     <h1>All users</h1>
 
     <%= will_paginate %>
 
-    <ul class="users">
-    <% @users.each do |user| %>
-        <%= render user %>
-    <% end %>
-    </ul>
+        <ul class="users">
+            <% @users.each do |user| %>
+                <%= render user %>
+            <% end %>
+        </ul>
 
-<%= will_paginate %>
\ No newline at end of file
+    <%= will_paginate %>
```

Comment:
@shaniajubelag Observe proper spacing. Indention is added when it is not needed, please indent backwards.